pyquil/external/rpcq.py

```diff
@@ -44,7 +44,7 @@ def dict(self, **kwargs: Any) -> Dict[str, Any]:
         exclude = kwargs.get("exclude") or set()
         if not self.dead:
             exclude.add("dead")
-        kwargs["exclude"] = {"dead"}
```

Comment:
FYR, this was a bug earlier and should have been `excluded` rather than `{"dead"}`, the motivating issue of this PR aside.